packages/modals/src/elements/TooltipModal/TooltipModal.tsx

```diff
@@ -49,49 +49,51 @@ interface IStaticTooltipModalExport<T, P>
 
 export interface ITooltipModalProps extends HTMLAttributes<HTMLDivElement> {
   /**
-   * The `HTMLElement` to position the `TooltipModal` relative to.
+   * Positions the modal relative to the provided `HTMLElement`
    */
   referenceElement?: HTMLElement | null;
   /**
-   * [Popper.js v2 modifiers](https://popper.js.org/docs/v2/modifiers/)
-   * to customize positioning logic.
+   * Modifies [Popper instance](https://popper.js.org/docs/v2/modifiers/) to customize positioning logic
    */
   popperModifiers?: Array<Partial<Modifier<any, any>>>;
   /**
-   * These placements differ from the default naming of Popper placements to help
-   * assist with RTL layouts.
+   * Provides placement names differing from the default Popper placement names to
+   * assist with RTL layout creation
```

Comment:
Oy, the previous description ends up being a red herring here. The update you made on `dropdowns` was excellent: https://github.com/zendeskgarden/react-components/pull/957/files#diff-8fc5bdeb0d77c73a3b03ce92de8df5ff9a393a3b092cc638a7700df6c8f42ec3L30-L33

Reply:
Seemed to have gone over my head this time around!